templates/feedback-en.html

```diff
@@ -8,8 +8,8 @@
 	"breadcrumb": [
 		{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" }
 	],
-	"css": ["demos/feedback/demo/feedback"],
-	"js": ["demos/feedback/demo/feedback"],
+	"css": ["https://wet-boew.github.io/v4.0-ci/demos/feedback/demo/feedback.min.css"],
+	"script": ["https://wet-boew.github.io/v4.0-ci/demos/feedback/demo/feedback.min.js"],
```

Comment:
Just wondering why GCWeb's ``js`` property was renamed to ``script``? WET 4.0 uses ``js`` and GCWeb used to use it too.

I don't really see what the point of renaming it was. IMO it makes GCWeb needlessly deviate from WET. Plus ``script`` is longer to write than ``js``.

Reply:
One or the other don't matter. Let's discuss at the next code sprint. But I know that if we wait and we have a few jekyll website that use gcweb-jekyll theme, that change might an issue.